### DISCUSSION TITLE
Add Label to SegmentedControl

### DISCUSSION THREAD

**Discussion by Crocsx** [mantinedev/mantine#5415] [Feature requests]
Hello,

I would suggest adding labels to SegmentedControl 

In the DOC there is some SegmentedControl with labels: 
https://mantine.dev/core/segmented-control/#transitions

I guess those are done using something like

```
        <Text size="sm" fw={500} mt={3}>
          Label
        </Text>
```

But this is annoying as we have to be sure that our `Text` match the `Label` style in the app. I think it's a normal use case to allow SegmentedControl to have or not labels.

**Comment by rtivital** [comment]
You can do that with Input.Wrapper or Input.Label components – https://mantine.dev/core/input/#inputwrapper-component
